webapp/components/Modal/ReportModal.vue

```diff
@@ -59,6 +60,9 @@ export default {
   },
   methods: {
     async cancel() {
+      if (this.callbacks.cancel) {
+        await this.callbacks.cancel()
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T12:59:03Z" title="Monday, June 3rd 2019, 2:59:03 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -59,6 +60,9 @@ export default {
   },
   methods: {
     async cancel() {
+      if (this.callbacks.cancel) {
+        await this.callbacks.cancel()
```


The `if` case would not be necessary if `cancel` would have the empty function as the default.